tests/Ormolu/HackageInfoSpec.hs

```diff
@@ -0,0 +1,472 @@
+module Ormolu.HackageInfoSpec where
+
+import qualified Data.Map.Strict as Map
+import Ormolu.Fixity
+import Test.Hspec
+
+(|->) :: a -> b -> (a, b)
```

Comment:
Can we get rid of this operator? Using tuples for this kind of purpose is idiomatic in Haskell, so a custom operator rather adds fiction than helps readability.